packages/preview/hhn-unitylab-thesis-template/0.0.2/typst.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "hhn-unitylab-thesis-template"
+version = "0.0.2"
+entrypoint = "src/config.typ"
+
+authors = ["@Ferrys93de"]
+license = "https://opensource.org/license/mit"
+description = "The official Thesis Template from the Usability and Interaction Technology Laboratory (UniTyLab) of the Heilbronn University (HHN)"
+
+homepage = "https://www.hs-heilbronn.de/en"
+repository = "https://github.com/Ferrys93de/UniTyLab-Thesis-Template"
+keywords = [
+    "hhn",
+    "hochschule heilbronn",
+    "heilbronn university",
+    "unitylab",
+    "thesis",
+    "documentation",
+]
+categories = ["thesis", "report"]
+exclude = [""]
```

Comment:
I think you should remove this line, otherwise the root of your package will match and everything will be excluded.